systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/InternalKafkaClient.java

```diff
@@ -312,6 +312,9 @@ public String getPodName() {
 
         boolean hasPassed = consumerGroups.run(timeoutMs);
         LOGGER.info("ConsumerGroups finished correctly: {}", hasPassed);
+        if (!hasPassed) {
+            throw new RuntimeException("VerifiableClient has failed. Check stderr for more details.");
```

Comment:
Is it possible that here the race condition can occur? Did you try to execute this test 4-5 times? It's passing?

Reply:
Thanks for the review @see-quick. 

Yes I did, and it looks fine. In ColdBackupScriptST I create my own instance of InternalKafkaClient, so I don't see any race condition. I also execute all acceptance STs locally. All green, apart a couple of errors due to timeouts. If you have any additional test in mind, please let me know and I'll run.

I see that there are other build errors here, but I don't think they are related to my changes. WDYT?

Reply:
Unit and integration test run on my machine:

```shell
$ mvn -e -V -B -Dmaven.javadoc.skip=true -Dsurefire.rerunFailingTestsCount=5 -Dfailsafe.rerunFailingTestsCount=2 install
...
[INFO] Reactor Summary for Strimzi - Apache Kafka on Kubernetes and OpenShift 0.27.0-SNAPSHOT:
[INFO] 
[INFO] Strimzi - Apache Kafka on Kubernetes and OpenShift . SUCCESS [  1.914 s]
[INFO] kafka-agent ........................................ SUCCESS [  0.929 s]
[INFO] mirror-maker-agent ................................. SUCCESS [  0.171 s]
[INFO] tracing-agent ...................................... SUCCESS [  0.158 s]
[INFO] test ............................................... SUCCESS [  0.351 s]
[INFO] crd-annotations .................................... SUCCESS [  0.686 s]
[INFO] crd-generator ...................................... SUCCESS [  3.022 s]
[INFO] api ................................................ SUCCESS [02:07 min]
[INFO] mockkube ........................................... SUCCESS [  3.728 s]
[INFO] config-model ....................................... SUCCESS [  0.648 s]
[INFO] config-model-generator ............................. SUCCESS [ 15.968 s]
[INFO] certificate-manager ................................ SUCCESS [  5.392 s]
[INFO] operator-common .................................... SUCCESS [02:48 min]
[INFO] topic-operator ..................................... SUCCESS [30:02 min]
[INFO] cluster-operator ................................... SUCCESS [07:31 min]
[INFO] user-operator ...................................... SUCCESS [ 11.128 s]
[INFO] kafka-init ......................................... SUCCESS [  2.499 s]
[INFO] systemtest ......................................... SUCCESS [  0.747 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  43:18 min
[INFO] Finished at: 2021-12-19T16:20:03+01:00
[INFO] ------------------------------------------------------------------------
```